http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -771,7 +772,7 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
             e.workspaceId,
             network.resourceId,
             DeleteControlledAzureResourceRequest(
-              WsmJobControl(WsmJobId(s"delete-networks-${ctx.traceId.asString.take(10)}"))
+              WsmJobControl(WsmJobId(s"delete-networks-${network.resourceId.value.toString.take(10)}"))
```

Comment:
opt: Could all of these maybe rely on passing an id to a little helper like `toJobId(jobName, resourceId`, since we're doing a little bit of special logic (first 10 digits)?